x/gov/types/expectedKeepers.go

```diff
@@ -30,6 +30,7 @@ type AuthKeeper interface {
 
 type PosKeeper interface {
 	RewardForRelays(ctx sdk.Ctx, relays sdk.BigInt, address sdk.Address)
+	RewardForRelaysPerChain(sdk.Ctx, string, sdk.BigInt, sdk.Address)
```

Comment:
NIT: Can you name the parameters? I'm assuming the second param is the chainID, but it's unclear.